docs/samples/calling/app.js

```diff
@@ -1301,6 +1301,10 @@ async function createCustomContact() {
       type: 'work',
       value: formData.get('phone')
     }],
+    emails: [{
```

Comment:
We weren't adding emails to a custom contact even though the samples page has a field for it. Fixed this issue.